### ISSUE TITLE
How to force Geany to open unknown files as plain text/ASCII?

### ISSUE THREAD

**Issue by that-ben** [geany/geany#3686]
Is there a way to force Geany to open ANY file it doesn't recognize as plain text or ASCII? I don't care for syntax on some of the files, I just want to see the raw contents.  NotePad++ didn't have any issue in opening whatever and displaying it (even binary files) so can Geany do that?

**Comment by elextr** [comment]
No, Geany cannot open files with a NUL byte in them, it is written in C where NUL means end of string.  Notepad++ is written in C++ which can handle strings with NUL in them.  Also Geany assumes that the buffer contents are UTF-8, so as a file is opened its encoding is converted to UTF-8.  So if it can't find an encoding that works it won't open the file.
